### PR TITLE
Qt 5 Support

### PR DESCRIPTION
**Pull request by fu5ha**
See https://github.com/appleseedhq/appleseed/pull/2509

### CODE REVIEW

File: src/SeExpr/CMakeLists.txt [hunk 1]
```diff
@@ -94,9 +95,3 @@ ENDIF()
 ## Traverse subdirectories
 ADD_SUBDIRECTORY (src/SeExpr)
 ADD_SUBDIRECTORY (src/SeExprEditor)
-ADD_SUBDIRECTORY (src/doc)
```
Comment (dictoon) [inline src/SeExpr/CMakeLists.txt]: Why did you remove those?

**Review by dictoon — COMMENTED** [review]
Amazing 😍

File: src/SeExpr/CMakeLists.txt (hunk 1 above)
Comment (fu5ha) [reply src/SeExpr/CMakeLists.txt]: Just because I didn't think they were used for our case and so they didn't need to be built

Comment (dictoon) [reply src/SeExpr/CMakeLists.txt]: Ok. I would prefer if we keep the CMake setup as untouched as possible. I also find it useful to build the docs, demos and tests.

**Comment by fu5ha** [comment]
Reverted and tested that it still builds the rest properly, which it does, and squashed commits. Think this is ready to go :)

**Review by dictoon — APPROVED** [review]
Brilliant, thank you!
